docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source.md

```diff
@@ -0,0 +1,107 @@
+---
+id: stix-taxii-2-client-source
+title: STIX/TAXII 2 Client Source
+sidebar_label: STIX/TAXII 2 Client
+tags:
+  - cloud-to-cloud
+  - stix
+  - taxii
+description: Learn how to setup a STIX/TAXII 2.x client to collect threat intelligence indicators into the Sumo Logic environment.
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/taxii-2/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/taxii-2/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/taxii-2/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+[STIX/TAXII](https://oasis-open.github.io/cti-documentation/) are two standards used together to exchange threat intelligence information between systems. STIX defines the format and structure of the data. TAXII defines how the API endpoints are served and accessed by clients. This Sumo Logic source supports collecting indicators from STIX/TAXII 2.0 and 2.1 versions. The legacy STIX/TAXII 1.x versions are not supported with this source.
+
+:::note
+This source is available in the [Fed deployment](/docs/api/getting-started#sumo-logic-endpoints-by-deployment-and-firewall-security).
+:::
+
+## Data collected
+
+This source collects threat intelligence indicators from a vendors STIX/TAXII 2.x endpoints. This means the specific endpoints we collect data from are the endpoints defined in the [TAXII standard](https://oasis-open.github.io/cti-documentation/taxii/intro). Vendor APIs must follow the standard. The source will collect all indicators from the TAXII server when it runs for the first time and it will check for updates once an hour. This one-hour polling interval can be adjusted in the source configuration.
+
+## Setup
+
+### Vendor configuration
+
+:::note
+The threat intel vendor must follow the STIX/TAXII 2.0 or 2.1 standard.
+:::
+
+1. Identify a vendor who uses the STIX/TAXII 2.0 or 2.1 standard for sharing threat intelligence indicators
+1. Follow their documentation to obtain the following information:
+  * The STIX/TAXII version they provide (either 2.0 or 2.1)
+  * The TAXII Discovery URL
+  * Your authentication credentials if required
```

Comment:
```suggestion
1. Follow their documentation to obtain the following information:
   * The STIX/TAXII version they provide (either 2.0 or 2.1)
   * The TAXII Discovery URL
   * Your authentication credentials if required
```

Reply:
The indenting wasn't working properly. Adding another space makes it line up correctly.